doc/stdlib/parallel.rst

```diff
@@ -374,6 +377,11 @@ General Parallel Computing Support
             body
         end
 
+.. function:: @everywhere
+
+    Executes an expression as is on all processors. In the event of exceptions on
```

Comment:
Grammatically I think "an exception on any" makes more sense than "exceptions on any"

Also what does "as is" mean here? Maybe a little more explanation would be good.